tsconfig.json

```diff
@@ -13,7 +13,8 @@
         "module": "commonjs",
         "outDir": "./out/",
         "skipLibCheck": true,
-        "target": "es2019"
+        "target": "es2019",
+        "esModuleInterop": true
```

Comment:
Isn't this a breaking change?

Either way, I believe it can be reverted when we drop the newly introduced dependency.